resources/assets/js/components/TableRow.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { Link, browserHistory } from 'react-router';
+import MyGlobleSetting from './MyGlobleSetting';
+
+
+class TableRow extends Component {
+  constructor(props) {
+      super(props);
+      this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
Why do you have to bind(this) ?

Reply:
Because my delete and edit buttons are in the <form> </form>